### PR TITLE
Perf parse params decode text

### PR DESCRIPTION
**Pull request by Uzlopak**
Based on the work of mscdex regarding the decodeText. Pretty smart how he did it. 

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by Eomm — APPROVED** [review]
> Based on the work of mscdex regarding the decodeText. Pretty smart how he did it.

Now sure to understand: did we charry-pick some work from the original busboy repo? - In case I would add credits

**Comment by kibertoad** [comment]
@Uzlopak Can you resolve the conflicts?

**Comment by kibertoad** [comment]
@Uzlopak what are the benchmarks agains the master?

**Comment by Uzlopak** [comment]
@kibertoad 
PTAL

**Comment by kibertoad** [comment]
@Uzlopak can you run benchmark to see the perf difference?

**Comment by Uzlopak** [comment]
@kibertoad 
I dont know... tbh photofinish is a little bit complicated. I maybe rewrite the new benchmark in tinybench. Or can you rewrite that benchmark from benchmark to photofinish?

File: bench/busboy-form-bench-latin1.js [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
-      busboy = new Busboy({
-        headers: {
-          'content-type': 'multipart/form-data; boundary=' + boundary
-        }
-      }),
-      buffer = createMultipartBufferForEncodingBench(boundary, 100, 'iso-8859-1'),
-      mb = buffer.length / 1048576;
+for (var i = 0, il = 10000; i < il; i++) { // eslint-disable-line no-var
+  const boundary = '-----------------------------168072824752491622650073',
+    busboy = Busboy({
```
[the remark this answers is not in the document]
Comment (Uzlopak) [reply bench/busboy-form-bench-latin1.js:9]: I have to check.

**Comment by kibertoad** [comment]
@Uzlopak We can rewrite to tinybench, photofinish main strength is that it allows running easily across Node versions, which is an overkill here.

**Comment by Uzlopak** [comment]
@kibertoad 
can we just merge? I have no motivation to rewrite the benchmarks. Maybe create an issue to rewrite the benchmarks and slab a 'good-first-issue' label to it.

**Pull request merged**